eventserver/eventServer.js

```diff
@@ -6,30 +6,28 @@ import createSubscriber from "pg-listen";
 const sseEmitter = new EventEmitter();
 
 const server = http.createServer((req, res) => {
-
   res.writeHead(200, {
     "Content-Type": "text/event-stream",
     "Cache-Control": "no-cache",
-    "Connection": "keep-alive",
+    Connection: "keep-alive",
```

Comment:
very nit-picky, so advance apologies, but Connection is unquoted in the new file here. Still works the same of course, just breaks the consistency of rest of rest of headers appearance.

Reply:
Thanks, no need to apologize, love the detail-orientedness. That was a prettier-driven change. When I added back in quotes and pressed save, prettier reverted it back to no quotes. Must be conventional to have connection be unquoted in that context.

Reply:
Oh I gotcha. Sounds like Prettier would unquote all of the object properties (at least the one word ones) if it had its druthers. Thanks for the cleanup work.
![image](https://github.com/Team3-Counterpoint/snowclone/assets/69982973/78b46e31-49c7-49b9-8cfa-b55609f11035)